pkg/spec/createconfig.go

```diff
@@ -145,6 +146,37 @@ func (c *CreateConfig) CreateBlockIO() (*spec.LinuxBlockIO, error) {
 	return c.createBlockIO()
 }
 
+// AddContainerInitBinary adds the init binary specified by path iff the
+// container will run in a private PID namespace that is not shared with the
+// host or another pre-existing container, where an init-like process is
+// already running.
+//
+// Note that AddContainerInitBinary prepends "/dev/init" "--" to the command
+// to execute the bind-mounted binary as PID 1.
+func (c *CreateConfig) AddContainerInitBinary(path string) error {
+	if path == "" {
+		return fmt.Errorf("please specify a path to the container-init binary")
+	}
+	if !c.PidMode.IsPrivate() {
+		logrus.Debugf("won't add init binary: PID namespace isn't private")
+		return nil
+	}
+	if c.Systemd {
+		return fmt.Errorf("cannot use init binary with systemd")
+	}
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return errors.Wrap(err, "container-init binary not found on the host")
```

Comment:
Add the `path` to the error message.

Reply:
That's already there as we're wrapping the `os.Stat()` error.

Reply:
I did not know that os.Stat error included the path.